src/process_controller.cpp

```diff
@@ -280,7 +275,7 @@ void ProcessController::handle_signal(pid_t child, int status)
         {
         case SIGSTOP:
         {
-            Log::debug() << "Set ptrace options for process: " << child;
+            Log::debug() << "Set ptrace options for " << child.name();
```

Comment:
seeing you writing child.name() for the hundredths time, why not define an `operator<<(std::ostream&, Thread|Process|CPU)` ?